git_machete/cli.py

```diff
@@ -99,6 +99,153 @@ def launch(orig_args: List[str]) -> None:
             f"Python {version_str} is no longer supported. Please switch to Python 3.6 or higher.\n")
         sys.exit(1)
 
+    def parse_options_argparse(in_args: List[str]) -> argparse.Namespace[str]:
```

Comment:
Remove `[str]` after `Namespace`, cause it breaks flow atm:
<img width="890" alt="Screenshot 2021-09-13 at 14 50 11" src="https://user-images.githubusercontent.com/73847352/133088003-fd140657-58c0-41eb-9724-4fb42d391f63.png">

Reply:
As I mentioned in the chat, it's not something that is supposed to be run or used in any sense just yet. I'm still choosing the place where particular code should be, I'm still testing functionalities.